squirrel_object_perception_msgs/srv/StartRobotTracking.srv

```diff
@@ -0,0 +1,5 @@
+# Start tracking the robots position
+# author: Edith Langer, langer@acin.tuwien.ac.at
+
```

Comment:
empty? is it right?

Reply:
Yes, that's right. It just triggers the tracker to start. You get the result when the stop-service is called.